### DISCUSSION TITLE
Where configure "normal power" and "measured power"

### DISCUSSION THREAD

**Discussion by alexojegu** [zwave-js/zwave-js-ui#3834] [Q&A] [answered]
I have a Aeotec Z-Stick 7, at [documentation](https://aeotec.freshdesk.com/support/solutions/articles/6000230552-z-stick-7-user-guide-) I can see:
> Best TX Power Levels.
>
> For US Frequency:
>  - TX Power Level Normal = 9.9 dBm (or 99 value)
>  - TX Power Level Measured = -6.5 dBm (or -65 value)
> 
> For EU Frequency:
> - TX Power Level Normal = 9.9 dBm (or 99 value)
> - TX Power Level Measured = -2.3 dBm (or -23 value)

But on Z-Wave JS UI I can found:
 - Normal Power Level at `Control Panel -> controller device` and at `Settings -> Z-Wave`
 - Measured Output Power at `Control Panel -> controller device` and at `Settings -> Z-Wave`

So, where is the right place to configure it? What is the difference?

I understand:
 - Aeotec TX Power Level Normal == Zwave JS UI Normal Power Level
 - Aeotec Measured Output Power == Zwave JS UI Measured Output Power

**Comment by kpine** [comment, marked as answer]
Control panel issues a single one time command.

Settings are applied every time ZUI restarts, so it persists it even after firmware upgrade or changed via control panel.

**Reply by alexojegu** [reply]
So both are the same, but the settings in the `Control Panel` may be lost when updating the firmware, while the settings in `Settings` will be forcibly applied on every Zwave JS startup.
